lib/tasks/bulk_import_documents_from_csv.rake

```diff
@@ -1,13 +1,20 @@
 desc "Import design decisions from a CSV file and save them with optional attachments"
 
 task :bulk_import_documents_from_csv, %i[csv_file_path] => :environment do |_, args|
+  require "json"
+
   csv_file_path = args[:csv_file_path]
 
   unless File.exist?(csv_file_path)
     puts "CSV file not found"
     exit
   end
 
+  schema_path = Rails.root.join("lib/documents/schemas/design_decisions.json")
+  schema = JSON.parse(File.read(schema_path))
```

Comment:
We should use the `FinderSchema.load_from_schema(type)` method here rather than hard-code the filepath and parse the schema manually.